controllers/baremetalset_controller.go

```diff
@@ -505,17 +505,17 @@ func (r *BaremetalSetReconciler) baremetalHostProvision(instance *ospdirectorv1b
 	}
 
 	sts = append(sts, userDataSt)
-	ipCidr := ipset.Status.HostIPs[bmhName].IPAddresses["ctlplane"] // We use ctlplane as the MgmtNetwork too for now
+	ipCidr := ipset.Status.HostIPs[bmhName].IPAddresses["ctlplane"]
 	ip, network, _ := net.ParseCIDR(ipCidr)
 	netMask := network.Mask
 
 	// Network data cloud-init secret
 
 	templateParameters = make(map[string]interface{})
-	templateParameters["MgmtIp"] = ip.String()
-	templateParameters["MgmtInterface"] = instance.Spec.MgmtInterface
-	templateParameters["MgmtGateway"] = ipset.Status.Networks["ctlplane"].Gateway
-	templateParameters["MgmtNetmask"] = fmt.Sprintf("%d.%d.%d.%d", netMask[0], netMask[1], netMask[2], netMask[3])
+	templateParameters["CtlplaneIp"] = ip.String()
+	templateParameters["CtlplaneInterface"] = instance.Spec.CtlplaneInterface
+	templateParameters["CtlplaneGateway"] = ipset.Status.Networks["ctlplane"].Gateway
+	templateParameters["CtlplaneNetmask"] = fmt.Sprintf("%d.%d.%d.%d", netMask[0], netMask[1], netMask[2], netMask[3])
```

Comment:
We'll need to also change https://github.com/stuggi/osp-director-operator/blob/ctl_network/templates/baremetalset/cloudinit/networkdata to match these

Reply:
good catch!

Reply:
done